src/main/java/software/amazon/event/kafkaconnector/EventBridgeSinkTask.java

```diff
@@ -81,10 +74,11 @@ public void put(Collection<SinkRecord> records) {
       try {
         batches.stream()
             .flatMap(batch -> eventBridgeWriter.putItems(batch).stream())
-            .filter(EventBridgeWriterRecord::hasError)
+            .filter(EventBridgeWriterResult::isFailure)
+            .map(EventBridgeWriterResult::failure)
             .forEach(this::handleFailedEntries);
         // only report successful putEvents calls
-        statusReporter.setSentRecords(eventBridgeWriterRecords.size());
+        statusReporter.setSentRecords(records.size());
```

Comment:
might be a quirk of work that's still in progress but either the doc is wrong or the line 81 is wrong w `records.size()`.

Reply:
Yes, it's a leftover from the original source and will be fixed.